packs/october.js

```diff
@@ -0,0 +1,39 @@
+const icon = `data:image/svg+xml,%3Csvg width='310' height='310' xmlns='http://www.w3.org/2000/svg' preserveAspectRatio='xMidYMid meet'%3E%3Cg%3E%3Ctitle%3Eoctober%3C/title%3E%3Crect fill='none' id='canvas_background' height='602' width='802' y='-1' x='-1'/%3E%3C/g%3E%3Cg%3E%3Ctitle%3ELayer 1%3C/title%3E%3Cg id='svg_1' fill='white' transform='translate(0,310) scale(0.10000000149011612,-0.10000000149011612) '%3E%3Cpath fill='%23de6c26' id='svg_2' d='m1350,3031c-142,-44 -349,-218 -499,-420c-293,-396 -455,-869 -437,-1281c8,-202 49,-355 144,-545c136,-274 408,-551 655,-669c141,-67 134,-69 141,28c3,44 10,324 16,621c27,1373 44,1760 82,1913c6,23 14,42 19,42c12,0 36,-91 49,-183c5,-43 10,-177 10,-298c0,-120 3,-219 7,-219c3,0 57,50 119,110c69,68 120,110 133,110c18,0 19,-3 10,-27c-12,-31 -79,-132 -191,-285l-78,-108l1,-312c1,-172 4,-374 8,-449l6,-137l110,126c140,160 351,371 395,396l33,19l-6,-32c-20,-98 -95,-207 -374,-543l-163,-196l0,-311c0,-183 4,-311 9,-311c28,0 193,64 268,105c138,73 238,150 383,295c157,157 244,274 334,452c205,400 210,803 16,1190c-178,356 -546,721 -878,869c-117,53 -246,73 -322,50z'/%3E%3C/g%3E%3C/g%3E%3C/svg%3E`;
+
+const UIStrings = {
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by removing unused CSS, in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'unused-css-rules': "Consider reviewing the [plugins](https://octobercms.com/plugins) loading unused CSS on your site. To identify plugins that are adding unnecessary CSS, try running [code coverage](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) in Chrome DevTools. You can identify the theme/plugin responsible from the URL of the stylesheet. Look out for plugins that have many stylesheets in the list which have a lot of red in code coverage. A plugin should only enqueue a stylesheet if it is actually used on the page.",
```

Comment:
* "that are adding" -> "that add", "try running" -> "run" (avoid passive tense)
* theme and plugins are different concepts in this CMS I assume, right?
* `URL of the stylesheet` -> `stylesheet URL`
* `Look out for plugins that have many stylesheets in the list which have a lot of red in code coverage.` -> `Look for plugins with many stylesheets with lots of red in code coverage.`
* Is `enqueue` a particular concept in this CMS, or would `add` work just as well?

Reply:
Done all, just not totally sure on this:

> theme and plugins are different concepts in this CMS I assume, right?